odra-macros/src/ast/host_ref_item.rs

```diff
@@ -258,6 +258,28 @@ mod ref_item_tests {
                 pub fn approve(&mut self, to: Address, amount: U256) {
                     self.try_approve(to, amount).unwrap()
                 }
+
+                pub fn try_airdrop(&self, to: odra::prelude::vec::Vec<Address>, amount: U256) -> Result<(), odra::OdraError> {
```

Comment:
So, the module interface defines a slice and HostRef gets `Vec`?

Reply:
Yes, a module gets slices, but if you call the contract you pass a vec. Getting a runtime arg you can't deserialize a slice but a vector only.